huaweicloud/services/as/data_source_huaweicloud_as_groups.go

```diff
@@ -247,7 +250,11 @@ func DataSourceASGroups() *schema.Resource {
 							Computed:    true,
 							Description: "The agency name.",
 						},
-						"tags": common.TagsSchema(),
+						"tags": {
+							Type:     schema.TypeMap,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
```

Comment:
```suggestion
"tags": {
							Type:     schema.TypeMap,
							Computed: true,
							Elem:     &schema.Schema{Type: schema.TypeString},
						},
```

"tags": common.TagsComputedSchema(),